wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -118,6 +118,19 @@
    */
   public static final String PERSIST_VOLUMES_ATTRIBUTE = "persistVolumes";
 
+  /**
+   * This attribute enables or disables merging plugins while provisioning a workspace. When
+   * enabled, the plugin broker will be configured to attempt to merge plugins where possible (when
+   * two or more plugins share the same image and do not have conflicting settings). Should be
+   * set/read from {@link WorkspaceConfig#getAttributes}.
```

Comment:
`@link WorkspaceConfig#getAttributes` that does not seem to be true... Should it be `Devfile#getAttributes`?

Reply:
P.S. Even if it's technically is read from WorkspaceConfig (because AFAIR we convert Devfile to WorkspaceConfig/Environment) I think it's better to reference Devfile#attributes, since it's how user is supposed to use it.

Reply:
I mostly followed what we have for `PERSIST_VOLUMES_ATTRIBUTE` above :D -- I'll fix it.